pkg/cmd/server/start/admission.go

```diff
@@ -74,7 +74,9 @@ var (
 		"PodNodeConstraints",
 		overrideapi.PluginName,
 		imagepolicy.PluginName,
-		"BuildDefaults",
+		// TODO: remove the log setting logic from the build defaulter and make this
+		// default off again.
+		//"BuildDefaults",
```

Comment:
Move this up to the `defaultOnPlugins` stanza.  This should have failed unit tests.  Did it pass?